Diagrams

TimotMar opened the pull request:
Correction and updating of all the diagrams

TimotMar committed:
Diagrams

Correction and updating of all the diagrams

eddymio commented:
A few corrections and a few more details are expected.
More precisely, 
- on the use case diagrams, often the use of extends would be more suitable than includes (ie : view article does not include the necessity to post a comment - no need of link here imo - access the blog does not include the fact of posting an article ... ) Please review each interaction.
- Now on the sequence diagram, we are missing details, it is too general, you could concentrate and may be divide this diagram in several to match specific use cases. For instance a sequence about the Post article Scenario showing both primary scenario and alternative ones : User asks the post form, the system shows it, the user submit the form, the system validates and submit it to the DB and notify the user. But if the form is not fully completed, ask the user to do it again -> loop.
Another diag about Comment lifecycle : a user post a comment, the system operates (you can provide more details...) , the admin sees the new comment , he clicks on delete OR (alt scenario) validate the comment...
- Finally the class diagram, you should review the type of arrows you are using (inheritance vs instance) and that will be fine !

TimotMar committed:
Sequence diagram

A sequenced diagramm a litlle more specific, showing the major uses of the website

TimotMar committed:
New modifications

New modifications about extends and includes in the use case system

TimotMar committed:
CSS + maiil fixing

The code is getting more integrated, more clear. 
Comments have been added, also the CSS to get something more structured. 
The class Post and Comment have been created.

eddymio commented:
Still a few corrections on the diagrams, mostly on the sequence one so we can have a clear understanding of the sequences.

TimotMar committed:
Diagrams changes

Some diagrams have been changed, so its more clear, especially the Sequence Diagrams that have been totally redo
